setup.py

```diff
@@ -45,9 +45,9 @@
         'scitools-iris>=2.2',
         'scikit-learn',
         'shapely',
+        'SkillMetrics',
```

Comment:
Can you please add the conda package for this to meta.yml?

Reply:
I did that but when I updated the environment the package was not working either. I had to manually pip install it to make the diagnostic work.

Reply:
@sloosvel this is fine, if you install the tool with `pip install -e '.[develop]'` that will install `SkillMetrics=1.1.7` from pypi and you can check with a test script eg
```
import skill_metrics
print(skill_metrics.__file__, skill_metrics.__version__)
```

Reply:
ie no need to update the environment, pip install in dev mode will do that for you for pypi packages :beer:

Reply:
Thanks for clarifying!